src/dbnode/client/write_test.go

```diff
@@ -210,10 +289,10 @@ func writeTestSetup(t *testing.T, writeWg *sync.WaitGroup) (*writeState, *sessio
 	enqueueWg.Wait()
 	require.True(t, s.state.topoMap.Replicas() == sessionTestReplicas)
 	for i := 0; i < s.state.topoMap.Replicas(); i++ {
-		completionFn(host, nil) // maintain session state
+		completionFn(hosts[i], nil) // maintain session state
```

Comment:
more a question: is `hosts[i]` always valid/defined here?

Reply:
yes.
All hosts are valid.